src/main/java/ParkingLot/VehicleParking.java

```diff
@@ -2,32 +2,33 @@
 
 import java.util.*;
 
-public class VehicleParking {
+class VehicleParking {
```

Comment:
it is better to specify access modifiers explicitly rather then use default one

Reply:
done

Reply:
same for VehicleParking class